src/runtime/TemporalObject.h

```diff
@@ -427,11 +432,22 @@ class TemporalDuration : public Temporal, private TemporalDate, private Temporal
 
 class TemporalInstant : public Temporal {
 public:
+    static const int64_t hourToNanosecond = 3600000000000;
+    static const int64_t minuteToNanosecond = 60000000000;
+    static const int64_t secondToNanosecond = 1000000000;
+    static const int64_t millisecondToNanosecond = 1000000;
+    static const int64_t microsecondToNanosecond = 1000;
```

Comment:
Since these values are not instance members of `TemporalInstant`,
please start each name with a capital letter.